dependabot_api.py

```diff
@@ -2,8 +2,6 @@
 import json
 import requests
 from addict import Dict
-
-
```

Comment:
> Imports should be grouped in the following order:
>  -  Standard library imports.
>  -  Related third party imports.
>  -  Local application/library specific imports.
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Thanks, I was missing the blank line.

Reply:
Nope, it's out of order too

Reply:
It should be: 
```python
import json

import requests
from addict import Dict

import config
```

Reply:
builtins
plugins
user

Reply:
A small thing, but a good standard to follow

Reply:
:+1: